server/sonar-web/src/main/js/apps/organizations/navigation/OrganizationNavigation.tsx

```diff
@@ -43,18 +43,27 @@ export class OrganizationNavigation extends React.PureComponent<Props, State> {
     error: ''
   };
 
-  async componentDidMount() {
+ componentDidMount() {
     this.mounted = true;
+    this.fetchNotificiations();
+  }
+
+  componentDidUpdate(prevProps:any, prevState:any) {
+    this.fetchNotificiations();  
+  }
+
+  async fetchNotificiations(){
     const {organization} = {...this.props}
     await getRawNotificationsForOrganization(organization.kee).then((data:any)=>{
       const notfication = data?.organization?.notifications?.[0];
       if(notfication?.type==="error"){
         this.setState({ error: notfication.message });
+      }else{
```

Comment:
Can you format the code properly. Look how the Sonarqube team does it, and use the same style.

Reply:
I have updated the PR. Could you re look into it.